src/main/java/life/qbic/portal/portlet/MetadataProperty.java

```diff
@@ -0,0 +1,58 @@
+package life.qbic.portal.portlet;
+
+import java.util.Objects;
+
+/**
+ * This class describes a property of metadata for an image
+ * @param <T> The type of the contained value
+ *
```

Comment:
Same as below, NIT: stick to our class header template

Reply:
You could also state that this is a DTO. Check out how we have done this in the data-model lib
https://github.com/qbicsoftware/data-model-lib/blob/feature/dtos_imaging/src/main/groovy/life/qbic/datamodel/dtos/imaging/parameters/electronmicroscopy/ElectronMicroscopyParameters.groovy

Reply:
Thanks for the template, I adapted the documentation in a new commit 👍